components/external-link/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
+
+/**
+ * Internal dependencies
+ */
+import Dashicon from '../dashicon';
+import './style.scss';
+
+function ExternalLink( { href, children, ...additionalProps } ) {
+	return (
+		<a { ...additionalProps } href={ href } target="_blank">
+			{ children }
+			<span className="new-window-icon">
```

Comment:
@iseulde 
This isn't needed, I'll update the PR to include it on the anchor tag instead and drop the span.